security-indicators/base/argo-workflows/download-package-template.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: download-package
+  labels:
+    app: thoth
+    component: download-package
+spec:
+  templates:
+    - container:
+        env:
+          - name: THOTH_WORKFLOW_TASK
+            value: download_package
+          - name: APP_SCRIPT
+            value: app.sh
+          - name: THOTH_PACKAGE_NAME
+            value: "{{inputs.parameters.THOTH_PACKAGE_NAME}}"
+          - name: THOTH_PACKAGE_VERSION
+            value: "{{inputs.parameters.THOTH_PACKAGE_VERSION}}"
+          - name: THOTH_PACKAGE_INDEX
+            value: "{{inputs.parameters.THOTH_PACKAGE_INDEX}}"
+        image: "workflow-helpers:test"
```

Comment:
This should probably be latest or something

Reply:
`image: workflow-helpers` only actually as we use the local image from the namespace